modules/core/src/main/java/org/apache/ignite/internal/visor/baseline/VisorBaselineTaskResult.java

```diff
@@ -75,14 +86,91 @@ public VisorBaselineTaskResult() {
         Map<String, VisorBaselineNode> map = new TreeMap<>();
 
         for (BaselineNode node : nodes) {
-            VisorBaselineNode dto = new VisorBaselineNode(node);
+
+            VisorBaselineNode dto = new VisorBaselineNode(node, Collections.emptyList());
 
             map.put(dto.getConsistentId(), dto);
         }
 
         return map;
     }
 
+    /**
+     * @param nodes Nodes to process.
+     * @return Map of DTO objects, with resolved ip->hostname pairs.
+     */
+    private static Map<String, VisorBaselineNode> toMapWithResolvedAddresses(Collection<? extends BaselineNode> nodes) {
+        if (F.isEmpty(nodes))
+            return null;
+
+        Map<String, VisorBaselineNode> map = new TreeMap<>();
+
+        for (BaselineNode node : nodes) {
+            Collection<VisorBaselineNode.ResolvedAddresses> addrs = new ArrayList<>();
+            if (node instanceof IgniteClusterNode) {
+                for (InetAddress inetAddress: resolveInetAddresses((ClusterNode)node)) {
```

Comment:
According to our guideline we shouldn't use braces for one line for and if statements.